rose/client/main.py

```diff
@@ -68,16 +70,40 @@ def send_message(self, msg):
         self.client.sendLine(str(msg))
 
 
+def load_driver_module(file_path):
+    """
+    Title Try to load the driver module from file
```

Comment:
Title is not needed, sorry if my example text was confusing.

Also, "Try to" is not needed. If may be good we had an interesting handling of errors, trying to load the driver in various ways, but we don't.